Sources/TensorFlow/Operators/LinearAlgebra.swift

```diff
@@ -128,6 +139,37 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
     }
 }
 
+/// Returns an identity matrix or a batch of matrices.
+///
+/// - Parameters:
+///   - rowCount: The number of rows in each batch matrix.
+///   - columnCount: The number of columns in each batch matrix.
+///   - batchShape: The leading batch dimensions of the returned tensor.
+/// TODO: Make columnCount and batchShape optional.
+public func eye<Scalar: Numeric>(
+    rowCount: Int,
+    columnCount: Int? = nil,
+    batchShape: [Int] = []
+) -> Tensor<Scalar> {
+    let columnCount = columnCount ?? rowCount
+    let diagonalSize = min(rowCount, columnCount)
+    var diagShape = [diagonalSize]
+    if(!batchShape.isEmpty) {
+        diagShape = batchShape + diagShape
+    }
+    let diagonalOnes = Tensor<Scalar>(ones: TensorShape(diagShape))
+    if rowCount == columnCount {
+        return diagonalOnes.diagonal()
+    } else {
```

Comment:
After an early `return` in an `if` statement, no `else` clause is necessary. Please move code to the same scope as the `if` statement to avoid unnecessary indentation.